packages/cli-repl/src/logger.ts

```diff
@@ -0,0 +1,162 @@
+import pino from 'pino'
+import path from 'path'
+import os from 'os'
+
+function logger (bus) {
+  const dest = path.join(os.homedir(), './.mongosh_log')
+  const log = pino({ name: 'monogsh' }, pino.destination(dest))
+
+  bus.on('*', function() {
```

Comment:
fancy!

Reply:
Maybe this is the difference :)

Reply:
Maybe is a stupid idea, but would it be possible to use this for everything? emitting all the infos as an object:

```
bus.emit('method:aggregate', { collection, pipeline });
```

Reply:
Yep, this is the difference! Nanobus has a star event that we can append stuff too that all events might get. 

I don't think it's good to use this for everything since we are going to be sending different information to telemetry and logging and it will be on case by case bases. For example, collection information should probably not be sent to telemetry, since that might conflict with GDPR.

Reply:
Makes sense!